client/src/pages/Settings/ProfilePhoto/ProfilePhoto.tsx

```diff
@@ -0,0 +1,95 @@
+import useStyles from './useStyles';
+import hero from '../../../images/landing/hero.jpg';
+import Button from '@mui/material/Button';
+import { Box, Typography } from '@mui/material';
+import SettingHeader from '../../../components/SettingsHeader/SettingsHeader';
+import { User } from '../../../interface/User';
+import { ChangeEvent, useState, useRef, useEffect } from 'react';
+import uploadPhoto from '../../../helpers/APICalls/uploadPhoto';
+import deletePhoto from '../../../helpers/APICalls/deletePhoto';
+import { useSnackBar } from '../../../context/useSnackbarContext';
+
+interface Props {
+  header: string;
+  currentUser?: User; // set to optional but always passed in from settings
+  currentProfile?: any;
```

Comment:
Let's use the Profile interface here instead of `any` (if we don't have one, maybe we can create it) 😄